### PR TITLE
build: update lnd version to v0.16.1

### PR DESCRIPTION
**Pull request by Roasbeef**

### CODE REVIEW

**Comment by Roasbeef** [comment]
Hmm proto compilation broke: https://github.com/lightninglabs/lightning-terminal/actions/runs/4748439253/jobs/8434636482?pr=532

Do I need to manually check in the protos somewhere?

**Comment by guggero** [comment]
You need to run `make protos` which updates the protos automatically in a way grpc-web needs.

File: go.mod
Comment (ellemouton) [inline go.mod:25]: think we wanna wait till 0.16.1-beta before doing a release here. 

i'm also happy to do the LiTd release btw 👍

**Comment by Roasbeef** [comment]
I ran `make protos` but it still looks borked? 

Feel free to take over this if you want @ellemouton, it's pushed to origin, so you should be able to directly modify the branch.

**Comment by ellemouton** [comment]
> I ran make protos but it still looks borked?

Yeah, it needed a manual entry in the `app/src/util/tests/sampleData.ts` file. All good now!

**Review by jamaljsr — APPROVED** [review]
tACK LGTM 👌

**Comment by guggero** [comment]
Will properly review and test tomorrow. But we should probably also include the just tagged Loop version `v0.22.1-beta`.

File: go.mod
Comment (guggero) [inline go.mod:20]: Sorry, I forgot that we also have a new version of Faraday (v0.2.11-alpha). Not super important but nice to have (just new REST mappings).

**Review by guggero — APPROVED** [review]
tACK, LGTM :tada:

**Comment by bhandras** [comment]
I wonder if we want to have MuSig2 loops by default with this release? cc @levmi

**Review by guggero — APPROVED** [review]

**Comment by ellemouton** [comment]
> I wonder if we want to have MuSig2 loops by default with this release?

Cool yeah will wait for confirmation from @levmi before merging & releasing

**Comment by ellemouton** [comment]
cool - have bumped loop to 0.23.0. gonna merge this once we have all green checks

**Pull request merged**
